openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx

```diff
@@ -76,7 +80,17 @@ const SchemaTable = ({
   tablePartitioned,
 }: SchemaTableProps) => {
   const { t } = useTranslation();
-
+  const { permissions } = usePermissionProvider();
+
+  const editDisplayNamePermission = useMemo(
+    () =>
+      checkPermission(
+        Operation.EditDisplayName,
+        ResourceEntity.TABLE,
+        permissions
+      ),
+    [permissions]
+  );
```

Comment:
Here we should use evaluated permissions, so use `getEntityPermissionByFqn` and get the permission of the table entity by fqn.

Reply:
Done